packages/react-component-playground/src/components/component-playground.jsx

```diff
@@ -43,10 +43,7 @@ module.exports = React.createClass({
     editor: React.PropTypes.bool,
     fixture: React.PropTypes.string,
     fullScreen: React.PropTypes.bool,
-    proxies: React.PropTypes.arrayOf(React.PropTypes.shape({
-      component: React.PropTypes.func.isRequired,
-      ref: React.PropTypes.string,
-    })),
+    proxies: React.PropTypes.arrayOf(React.PropTypes.func),
```

Comment:
Yes, they do. Wasn't sure about ES6 classes but it seems [they do too](http://jsfiddle.net/eb8wbwsk/).